var/spack/repos/builtin/packages/sprng/package.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Sprng(AutotoolsPackage):
+    """SPRNG: A Scalable Library For Pseudorandom Number Generation
+       Sprng is a distributed process-aware random number generator that
+       avoids correlations in random number sequences across processes.
+    """
+
+    maintainers = ['kayarre']
+
+    homepage = "http://www.sprng.org"
+    url      = "http://www.sprng.org/Version5.0/sprng5.tar.bz2"
+
+    version('5.0',
+            sha256='9172a495472cc24893e7489ce9b5654300dc60cba4430e436ce50d28eb749a66')
+
+    variant('mpi', default=True, description='Enable MPI support')
+    variant('fortran', default=False, description='Enable Fortran support')
+
+    depends_on('mpi+cxx', when='+mpi')
+
+    def configure_args(self):
+        configure_args = []
+        configure_args += self.enable_or_disable('mpi')
+        configure_args += self.enable_or_disable('fortran')
+        # if '+fortran' in self.spec:
+        #     configure_args.append('--with-fortran')
+
+        if '+mpi' in self.spec:
+            # configure_args.append('--with-mpi')
+            if self.spec.satisfies('^openmpi'):
+                configure_args.append('LIBS=-lmpi_cxx')
+            configure_args.append('CC={0}'.format(self.spec['mpi'].mpicc))
+            configure_args.append('CXX={0}'.format(self.spec['mpi'].mpicxx))
+            if '+fortran' in self.spec:
+                configure_args.append('FC={0}'.format(self.spec['mpi'].mpifc))
+        return configure_args
+
+    @run_after('build')
+    @on_package_attributes(run_tests=True)
+    def check_install(self):
```

Comment:
This part is probably worth a TODO or a FIXME comment, since it will likely fit in the kind of tests that @becker33 is enabling for packages in #15702

Reply:
is it sufficient to to name the new feature?